### PR TITLE
Added instructions for enabling Istio CNI on k3s

### PR DESCRIPTION
**Pull request by dgiebert**

### CODE REVIEW

**Comment by martyav** [comment]
These instructions are so similar to the ones on https://ranchermanager.docs.rancher.com/integrations-in-rancher/istio/configuration-options/install-istio-on-rke2-cluster, that I think we should update that page (and the page title for it) instead. We can format the code block with tabs to display either RKE2 or K3s YAML.

**Comment by martyav** [comment]
Docusaurus tabs syntax: https://docusaurus.io/docs/markdown-features/tabs

**Comment by dgiebert** [comment]
This is a very good idea; I will implement it accordingly!

File: docs/integrations-in-rancher/istio/configuration-options/install-istio-on-rke2-cluster.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Additional Steps for Installing Istio on an RKE2 Cluster
+title: Additional Steps for Installing Istio on an RKE2 and K3s Clusters
```
Comment (martyav) [inline docs/integrations-in-rancher/istio/configuration-options/install-istio-on-rke2-cluster.md:2]:
```suggestion
title: Additional Steps for Installing Istio on RKE2 and K3s Clusters
```

**Review by martyav — CHANGES_REQUESTED** [review]
I checked to see if the tabs work correctly in a preview build, and they do.

File: docs/integrations-in-rancher/istio/configuration-options/install-istio-on-rke2-cluster.md [hunk 2]
```diff
@@ -8,30 +8,53 @@ When installing or upgrading the Istio Helm chart through **Apps,**
...
+          patches:
+          - path: spec.template.spec.containers.[name:install-cni].securityContext.privileged
+            value: true
+  values:
+    cni:
+      cniBinDir: /opt/cni/bin
+      cniConfDir: /etc/cni/net.d
+```
+</TabItem>
+<TabItem value="k3s">
```
Comment (btat) [inline docs/integrations-in-rancher/istio/configuration-options/install-istio-on-rke2-cluster.md:35]:
```suggestion
<TabItem value="K3s">
```

**Review by btat — CHANGES_REQUESTED** [review]

**Review by martyav — APPROVED** [review]

**Review by btat — APPROVED** [review]

**Pull request merged**